components/camel-vertx-kafka/camel-vertx-kafka-component/src/main/java/org/apache/camel/component/vertx/kafka/operations/VertxKafkaProducerOperations.java

```diff
@@ -189,4 +190,17 @@ private Object getMessageValue(final Message message, final Object inputData) {
         return VertxKafkaTypeSerializer.tryConvertToSerializedType(message, inputData,
                 configurationOptionsProxy.getValueSerializer(message));
     }
+
+    private Long getOverrideTimestamp(final Message message) {
+
+        Object timeStamp = configurationOptionsProxy.getOverrideTimestamp(message);
+        Long overrideTimestamp = 0L;
```

Comment:
in case if no `OVERRIDE_TIMESTAMP` being set, does it make sense to initialize it with `null` instead of `0L`? For me 0L is manipulating the timestamp in case there is no a timestamp being set.

Reply:
Yes I think it also should be null in this case.

Reply:
Done